README.md

```diff
@@ -1,5 +1,130 @@
+
+Based on this guidance, our project generated the following generalized defintion for "anonymization":
+
+> A process that removes the association between the identifying dataset and the data subject.
+
+#### Aggregation
+
+The concept of aggregated data bridges both the annonymity of information and the amount of information collected. In essence, it the combination of volume plus annonymity that allows aggregated data to be less prone to privacy concerns. The CCPA directly recognizes the concept of aggregation as follows:
+
+> information that relates to a group or category of consumers, from which individual consumer identities have been removed, that is not linked or reasonably linkable to any consumer or household, including via a device. “Aggregate consumer information” does not mean one or more individual consumer records that have been de­identified. [4]
+
+Additionally, the GDPR does recognize the concept of statistical research in Recital 162, stating:
+
+> Statistical purposes mean any operation of collection and the processing of personal data necessary for statistical surveys or for the production of statistical results. Those statistical results may further be used for different purposes, including a scientific research purpose. The statistical purpose implies that the result of processing for statistical purposes is not personal data, but aggregate data, and that this result or the personal data are not used in support of measures or decisions regarding any particular natural person.
+
+Based on these resources, our project generated the following generalized definition of "aggregation":
+
+> Information that relates to a group or category of individuals, from which individual identities have been removed, that is not linked or reasonably linkable to any individual.
+
 ### Repository for Tools and Documents for Hackathon Project
+
+### Future Work
+
+Through this initial project, our team identified the following areas for future work to continue to build out effective solutions to balance the accuracy of location-data and privacy concerns:
+
+1. Identifying the Privacy Risks 
+
+2. Further define key legal terms & factors with corresponding statistical analysis
+
+### Appendices 
+
+[1] Regulation (EU) 2016/679 of the European Parliament and of the Council of 27 April 2016 on the protection of natural persons with regard to the processing of personal data and on the free movement of such data, and repealing Directive 95/46/EC.
+
+[2] NIST Special Publication 800-53 (Rev. 4).
+
+[3] European Data Protection Board Guidelines 04/2020 onthe use of location data and contact tracing tools in the context of the COVID-19 outbreak, 2020.04.21.
+
+[4] The California Consumer Privacy Act of 2018, Cal. Civ. Code § 1798.100, <em>et seq.</em>
```

Comment:
Add:

Harvey, S. J. (2013). Smart Meters, Smarter Regulation: Balancing Privacy and Innovation in the Electric Grid. UCLA L. Rev., 61, 2068.

Fienberg, S. E. (2013). Is the Privacy of Network Data an Oxymoron?. Journal of Privacy and Confidentiality, 4(2).

Karwa, V., & Slavković, A. B. (2012, September). Differentially private graphical degree sequences and synthetic graphs. In International Conference on Privacy in Statistical Databases (pp. 273-285). Springer, Berlin, Heidelberg.

Li, B., Karwa, V., Slavković, A., & Steorts, R. C. (2018). A Privacy Preserving Algorithm to Release Sparse High-dimensional Histograms. Journal of Privacy and Confidentiality, 8(1).

Stutzman, F. D., Gross, R., & Acquisti, A. (2013). Silent listeners: The evolution of privacy and disclosure on Facebook. Journal of privacy and confidentiality, 4(2), 2.

Norton Rose Fulbright (2020). Contact tracing apps: A new world for data privacy.  Accessed on May 17, 2020 from https://www.nortonrosefulbright.com/en/knowledge/publications/d7a9a296/contact-tracing-apps-a-new-world-for-data-privacy

Tsang, Samantha (2020).  Here are the contact tracing apps being deployed around the world.  Accessed on May 20, 2020 from https://iapp.org/news/a/here-are-the-contact-tracing-apps-being-employed-around-the-world/